syntax = "proto3";

option csharp_namespace = "OddbitAi.Whisper";

message ProcessAudioRequest {
    bytes audioData = 1;
};

message ProcessAudioReply {
    int32 result = 1;
}

service WhisperService {
    rpc ProcessAudio (ProcessAudioRequest) returns (ProcessAudioReply);
}